backend/de.metas.manufacturing/src/main/java/org/eevolution/productioncandidate/service/ResourcePlanningPrecision.java

```diff
@@ -63,14 +60,16 @@ else if (codeNorm.equals(MINUTE_15.getCode()))
 	}
 
 	@NonNull
-	public Timestamp roundDown(@NonNull final Timestamp initialDate)
+	public ImmutableSet<String> getMinutes()
```

Comment:
what's the point of returning as strings here? why not ints?

Reply:
changed to Set<Integer>